product/domain/productBasics.go

```diff
@@ -90,6 +90,7 @@ type (
 		SaleableTo      time.Time
 		ActivePrice     PriceInfo
 		AvailablePrices []PriceInfo
+		LoyaltyPrice    LoyaltyPriceInfo
```

Comment:
Why do we need this addition? Is it the active loyalty price? Would call it then like this.

Reply:
done